GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -229,7 +229,7 @@ private void FormCommit_Load(object sender, EventArgs e)
             ResetUnStaged.Visible = AppSettings.ShowResetUnstagedChanges;
             CommitAndPush.Visible = AppSettings.ShowCommitAndPush;
             AdjustCommitButtonPanelHeight();
-	    showUntrackedFilesToolStripMenuItem.Checked = !Module.RunGitCmd("config status.showUntrackedFiles").Trim().Equals("no");
+            showUntrackedFilesToolStripMenuItem.Checked = !Module.EffectiveConfigFile.GetValue("status.showUntrackedFiles").Equals("no");
```

Comment:
Better use string.Equals(val, "astringvalue", StringComparison.OrdinalIgnoreCase) to ignore case